lib/trace.js

```diff
@@ -210,6 +212,18 @@ const _resolveDep = async ({
   bailOnMissing,
   addRootPackagePaths = false
 }) => {
+  // Support advanced Node.js ESM import types by ignoring built-in's as not
+  // needing any further inspection / inclusion.
+  if (IGNORE_IMPORT_PREFIXES.find((prefix) => depName.startsWith(prefix))) {
```

Comment:
I switched from `indexOf() !== -1` to `startsWith()` to be a little bit more of a semantic match with the fact that we're only matching prefixes, not string tokens anywhere throughout the target string.

Reply:
Thanks - it's indeed more performant.